server/services/emailService.js

```diff
@@ -0,0 +1,33 @@
+import nodemailer from "nodemailer"
```

Comment:
Should use the firebase email service

Reply:
Will also reach out to DishZero regarding aquiring a more official email, but that can occur later

Reply:
Removed nodemailer. Will use the firebase email service once obtain details regarding the sender's email address.